Dev branch

SantosVicente opened the pull request:
aceita ai man

SantosVicente committed:
chore: Add CRUD operations for transactions in PHP

- Implemented POST route for transaction creation.
- Modified PUT route to allow updating specific fields based on user input.
- Added DELETE route to remove transactions by transaction_id.
- Added the transactions table to the database and removed the queries table.

SantosVicente committed:
feat: Implemented logic to display charts on the home page and PHP route for GET transactions

- Added PHP code to fetch user transactions from the database
- Integrated Chart.js into the HTML file for chart creation
- Created a JavaScript script to generate bar and pie charts with dummy data
- Structured the HTML page with specific areas for charts and user actions
- Applied basic styling with CSS for a more pleasant visualization

SantosVicente committed:
fix: removing home script from index.html

vct-h-dias reviewed (approved):
LGTM (joia (preguiça de pegar emoji))

SantosVicente committed:
feat: dashboard implementation

- integration of update routes with the UI

- integration of delete routes with the UI

fix: remove the create route for the UI

- Months are created automatically when registering a user

Co-authored-by: Victor Hugo <vct-h-dias@users.noreply.github.com>
